File: JavaPoetDemo/app/src/main/java/com/example/javapoet/MainActivity.java
package com.example.javapoet;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.annotation.ContentView;
import com.example.core.ViewBind;


/**
 * @author tengfei
 */

@ContentView(id = R.layout.activity_main)
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ViewBind.bind(this);
    }


}

File: MVVMFramWork/app/src/main/java/com/example/tengfei/mvvm/MainActivity.kt
package com.example.tengfei.mvvm

import android.databinding.DataBindingUtil
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.bumptech.glide.Glide
import com.example.tengfei.mvvm.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var activityMainBinding:ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)
        val user = User()
        user.name = "fasf"
        user.header="http://k.zol-img.com.cn/sjbbs/7692/a7691515_s.jpg"
        activityMainBinding.user = user
        Glide.with(activityMainBinding.mainImage.context).load("http://k.zol-img.com.cn/sjbbs/7692/a7691515_s.jpg").into(activityMainBinding.mainImage)

       // user.name = "flalalafaf"
    }
}
